ntbs-service/Pages/Notifications/Denotify.cshtml

```diff
@@ -111,7 +111,7 @@
         </nhs-fieldset>
     </nhs-form-group>
 
-    <button nhs-button-type="Standard" asp-page-handler="Confirm">Confirm denotification</button>
+    <button id="confirm-dentoification-button" nhs-button-type="Standard" asp-page-handler="Confirm">Confirm denotification</button>
```

Comment:
fingers too fast for you on this one - it is correctly referenced in the ui test though!